.circleci/config.yml

```diff
@@ -163,13 +163,27 @@ jobs:
       - run:
           name: Run migrations ci
           command: yarn run db:migrate:ci
+      - run:
+          name: Install Headless Chrome dependencies
+          command: |
+            sudo apt-get install -yq \
+            gconf-service libasound2 libatk1.0-0 libatk-bridge2.0-0 libc6 libcairo2 libcups2 libdbus-1-3 \
+            libexpat1 libfontconfig1 libgcc1 libgconf-2-4 libgdk-pixbuf2.0-0 libglib2.0-0 libgtk-3-0 libnspr4 \
+            libpango-1.0-0 libpangocairo-1.0-0 libstdc++6 libx11-6 libx11-xcb1 libxcb1 libxcomposite1 libxcursor1 \
+            libxdamage1 libxext6 libxfixes3 libxi6 libxrandr2 libxrender1 libxss1 libxtst6 ca-certificates \
+            fonts-liberation libappindicator1 libnss3 lsb-release xdg-utils wget
```

Comment:
I can't think of a way to do this faster. The way this is specified here the `Install Headless Chrome dependencies` step is only run once per workflow.  Even if we could move it to an executor, and I don't think we can, it would still run once every time the workflow is run.  Actually, I'm shocked by how speedy this is. This step completes in 13 seconds.

Reply:
oh wow guess I should have looked at how long it took, I just assumed it would take a while :+1: